CHANGELOG.md

```diff
@@ -1,13 +1,18 @@
 # Changelog
 
+## Unreleased
+
+### Features
+
+- Implement local scope by adding overloads to the capture methods that accept a ScopeCallback ([#2075](https://github.com/getsentry/sentry-java/pull/2075))
+
 ## 6.0.0
 
 ### Sentry Self-hosted Compatibility
 
 - Starting with version `6.0.0` of the `sentry` package, [Sentry's self hosted version >= v21.9.0](https://github.com/getsentry/self-hosted/releases) is required or you have to manually disable sending client reports via the `sendClientReports` option. This only applies to self-hosted Sentry. If you are using [sentry.io](https://sentry.io), no action is needed.
 
 ### Features
-
```

Comment:
I'd rather keep a line break after the headers

Reply:
Sorry, wasn't intentional :) new line is back

Reply:
I think there were 2 before all the merging happened.